doc/index.doc

```diff
@@ -122,7 +122,7 @@ The third part provides information for developers:
 \addindex license
 \addindex GPL
 
-Copyright &copy; 1997-\showdate "%Y" by
+Copyright &copy; 1997-\showdate "%Y" by 
 <a href="mailto:doxygen@gmail.com">Dimitri van Heesch</a>.<p>
```

Comment:
Thanks, though better to join lines 126 and 127 as otherwise some editors / automatic scripts might remove the space later on again.